sensor_msgs/NavSatFix datum
geometry_msgs/Point point
---
sensor_msgs/NavSatFix llh
